### DISCUSSION TITLE
制作整合包

### DISCUSSION THREAD

**Discussion by rjdxxxxxccc** [Meloong-Git/PCL#4432] [提问] [answered]
这个启动器怎么制作整合包

**Comment by hejiehao** [comment, marked as answer]
可以到 `更多` -- `帮助` -- `指南` -- `整合包制作指南` 中查看教程
![image](https://github.com/user-attachments/assets/23ae8073-5d39-4d4d-9947-9bccc570d036)

**Comment by Chiloven945** [comment]
如果您指导出相关整合包文件的话可以在 Discussion 进行投票，目前也已经有相关pr。
